### PR TITLE
removed unnecessary texts and fixed correct option for qn8

### PR DESCRIPTION
**Pull request by c99SRS**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md)

### DOD

- [ ] I have added new quiz{'s}
- [ ] I have added new reference link{'s}
- [x] I have made small corrections/improvements

### Changes / Instructions

removed unnecessary texts and corrected the answer for maven Qn-8

### CODE REVIEW

**Comment by c99SRS** [comment]
@Ebazhanov made the necessary changes. Please check it and merge my PR. if any issues let me know, happy to fix them. Thanks!!

File: maven/maven-quiz.md [hunk 1]
```diff
@@ -52,16 +52,10 @@
 #### Q8. Which command is used to run the clean lifecycle followed by verify, install, and package with Maven?
 
 - [ ] `mvn clean install package`
```
Comment (haxkd) [inline maven/maven-quiz.md:54]: answer should be "mvn clean verify install package"

**Review by haxkd — CHANGES_REQUESTED** [review]

File: maven/maven-quiz.md (hunk 1 above)
Comment (Ebazhanov) [reply maven/maven-quiz.md:54]: agree, chatgpt said to me the same 
image

Comment (c99SRS) [reply maven/maven-quiz.md:54]: Thanks, @Ebazhanov @haxkd. making necessary changes

**Pull request merged**